### PR TITLE
Fix incorrect script reference in TRACING.md

### PR DESCRIPTION
**Pull request by YchauWang**
fixed the "Guest journal redirector" part of TRACING.md error, from "vsock-to-udp-server.sh" to "vsock-to-text-server.sh"

### CODE REVIEW

**Comment by caoruidong** [comment]
Thx. Please sign off your commit

**Comment by YchauWang** [comment]
#644

**Comment by jodh-intel** [comment]
Hi @YchauWang - thanks for raising this! The Travis CI is failing due to the format of your commit though:

```
Found 1 commit between commit 3317c4c6fe89fbf43bb0bef3b848bffd5cdd3b93 and branch master
ERROR: Commit 3317c4c6fe89fbf43bb0bef3b848bffd5cdd3b93: Failed to find subsystem in subject: "Update TRACING.md"
ERROR: checkcommits failed. See the document below for help on formatting
commits for the project.
```

This is due to our [`checkcommits`](https://github.com/kata-containers/tests/tree/master/cmd/checkcommits) tool which checks the format of commits.

There are three simple things you need to do:

- Add `docs:` as a prefix to the first line of your commit.
- Add a reference to a GitHub issue this PR fixes (I've just raised #646 for you to use).
- Add a `Signed-off-by:` comment.

In summary, your commit could look something like this to keep Travis happy:

```
docs:  Fix incorrect script reference in TRACING.md

Fixed the "Guest journal redirector" part of `TRACING.md` error from `vsock-to-udp-server.sh` to `vsock-to-text-server.sh`.

Fixes: #646 

Signed-off-by: your name <your-email-address@somewhere.com>
```

See https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#patch-format for further details.

**Comment by YchauWang** [comment]
@jodh-intel thank you very much for your subject. This is my first github commit, so sorry to make this mistake.

**Comment by chavafg** [comment]
@YchauWang can you please rebase with latest changes so that travis can pass? 
Thanks.

**Comment by YchauWang** [comment]
@chavafg Thanks, I have rebased the patch, the travis passed now.
